src/main/plugin-metadata/plugin-security.policy

```diff
@@ -4,7 +4,9 @@ grant {
   permission java.security.SecurityPermission "insertProvider.BC";
   permission java.security.SecurityPermission "removeProviderProperty.BC";
   permission java.util.PropertyPermission "jdk.tls.rejectClientInitiatedRenegotiation", "write";
+  permission java.security.SecurityPermission "getProperty.org.bouncycastle.rsa.max_size";
```

Comment:
Should be covered by `permission java.security.SecurityPermission "getProperty.*";`

Reply:
fixed